assets/prometheus-k8s/rules.yaml

```diff
@@ -334,6 +334,14 @@ spec:
     - expr: sum(rate(container_cpu_usage_seconds_total{container_name!="POD",container_name!="",pod_name!=""}[5m]))
         / sum(machine_cpu_cores)
       record: cluster:container_cpu_usage:ratio
+    - expr: sum(node:node_cpu_utilisation:avg1m)
```

Comment:
This would give us the sum of average CPU utilization in %. The sum of multiple % is not accurate out of 100. We need to use the raw data and not aggregated by node.